tests/unit/Imports/ProductImportTest.php

```diff
@@ -155,6 +157,194 @@ function (MockInterface $module) {
         $this->assertCount(1, $variableProduct->get_upsell_ids(), '1 upsell IDs should be set');
     }
 
+    public function dataProviderTestGetStockProperties()
```

Comment:
I made lately https://github.com/storekeeper-company/api-helper-scripts/blob/master/src/ExportChannelProductsToWooCommerceFormatCommand.php#L47 which contains all the logic. 
I needed it for compasison

Reply:
Changes made based on scenarios tested and discussed. See PR description for details.